app/_hub/kong-inc/ai-proxy/how-to/llm-provider-integration-guides/_huggingface.md

```diff
@@ -0,0 +1,87 @@
+---
+nav_title: Hugging Face
+title: Set up AI Proxy with Hugging Face
+minimum_version: 3.9.x
+---
+
+This guide walks you through setting up the AI Proxy plugin with [Hugging Face](https://huggingface.co/).
+
+{% include_cached /md/plugins-hub/ai-providers-prereqs.md snippet='intro' %}
+
+## Prerequisites
+
+{% include_cached /md/plugins-hub/ai-providers-prereqs.md snippet='service' provider='Hugging Face' %}
+* Hugging Face access token with permissions to make calls to the Inference API
+* [Text-generation model](https://huggingface.co/models?pipeline_tag=text-generation&sort=trending) from Hugging Face
+
+## Provider configuration
+
+### Set up route and plugin
+
+{% navtabs %}
+{% navtab Kong Admin API %}
```

Comment:
Why do we have the proxy advanced guides in `plugin_example` format, but the AI Proxy ones are written manually? I guess that's all of them, isn't it. Note to self to convert these examples after 3.9.

Reply:
I followed the same structure as for the other providers, but I asked myself the same question